sync/sync.go

```diff
@@ -57,15 +58,23 @@ func (s *Synchronizer) fetcherTask(ctx context.Context, height uint64, verifiers
 			if err != nil {
 				continue
 			}
+			declaredClasses := make(map[felt.Felt]*core.Class, len(stateUpdate.StateDiff.DeployedContracts))
```

Comment:
```go
len(stateUpdate.StateDiff.DeployedContracts) != len(stateUpdate.StateDiff.DeclaredClasses)
```

Therefore, just make it the following:
```suggestion
			declaredClasses := make(map[felt.Felt]*core.Class)
```

Reply:
Thanks! I think that was a typo on my end. It should be `len(stateUpdate.StateDiff.DeclaredClasses)`. Changed in most recent rebase.